### DISCUSSION TITLE
Smaller Queries

### DISCUSSION THREAD

**Discussion by myztillx** [pocketbase/pocketbase#1492] [Q&A]
New to pocket base and loving it so far! 

I was looking for a way to return less data (table columns) from a query. Right now I am using a separate table and using expanded, but making 2 queries each time I update feels odd. I did not see how to do this in the documentation. 

Ex. Table columns: id, updated, created, name, description, notes, archived, asset_number. 

For a specific query on a page I only care about: id and name. Getting the rest of the data at this time is not a big deal, since this is a personal app, but if I use it on a different project with lots of rows, that seems like a lot of unnecessary data.

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand what do you mean.

If you are asking for a way to return only a subset of collection fields when querying records (eg. `?fields=id,name`), this currently is not supported but it is in my todo and eventually could be added sometime in the future (you can also check this other older discussion https://github.com/pocketbase/pocketbase/discussions/354).

**Reply by myztillx** [reply]
That is what I was asking about, thank! I will keep an eye on the roadmap :smile:.
